api.bs

```diff
@@ -727,22 +727,28 @@ The arguments to <a method for=PrivateAttribution>measureConversion()</a> are as
 
 ### Operation ### {#measure-conversion-api-operation}
 
+Given <a dictionary lt=PrivateAttributionConversionOptions>|options|</a>:
+
 1.  Collect the implicit API inputs from the [=relevant settings object=]:
-    1.  The timestamp is set to the [=current high resolution time=].
-    2.  The [=conversion site=] is set to the result of
+    1.  Let |now| be the [=current high resolution time=].
+    1.  Let |topLevelSite| (the [=conversion site=]) be the result of
         [=obtain a site|obtaining a site=] from the [=top-level origin=].
-    3.  The [=intermediary site=] is set to
+    1.  The [=intermediary site=] is set to
         1.   a value of `undefined` if the [=origin=] is [=same site=]
              with the [=top-level origin=],
         1.   otherwise, the result of
              [=obtain a site|obtaining a site=] from the [=origin=].
+        <!-- TODO: the intermediary site is currently unused -->
```

Comment:
I'm not sure we even need to capture the intermediary site here. I would suggest just deleting this and filing an issue.

Reply:
I think we should file an issue to iron it out. At least what was presented in https://github.com/patcg/meetings/blob/main/2024/11/12-telecon/Delegation%20v2%20-%20PAT-CG.pdf indicated to me that the intermediary site in the stored impression was just a "filterable" value.

Reply:
Filed #102 (and we also have #55).